src/Splash.tsx

```diff
@@ -36,8 +36,6 @@ type Props = {
   isReady: boolean
 }
 
-SplashScreen.preventAutoHideAsync().catch(() => {})
```

Comment:
After updating, I was getting a `cannot read catch of undefined` error. This is already called in `App.native.tsx`. Removing this fixed 🤷